trace/trace.go

```diff
@@ -100,15 +138,15 @@ func (t *Trace) AddField(key string, val interface{}) {
 // The serialized form may be passed to NewTrace() in order to create a new
```

Comment:
this comment doesn't line up with reality anymore, does it?

actually - what calls this function now - feels superfluous now that we have propagators?

Reply:
Ah, I love it when there's code that can be deleted. Yup - removed.